src/convertToObject.js

```diff
@@ -12,7 +12,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  const properties = sourceString.split(';');
+
+  let pair;
+
+  for (let i = 0; i < properties.length; i++) {
+    if (properties[i].trim().length === 0) {
+      continue;
+    }
+
+    pair = properties[i].split(':');
+    pair[0] = pair[0].trim();
+    pair[1] = pair[1].trim();
+
+    result[pair[0]] = pair[1];
```

Comment:
You can simplify this section with destructuring:
```suggestion
    const [property, value] = properties[i].split(':');

    result[property.trim()] = value.trim();
```